service/src/main/resources/logback.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--We need this file to override the terra-commons-lib configuration for human-readable logging-->
+<configuration scan="true" scanPeriod="30 seconds">
+
+    <variable name="LOG_PATH" value="logs"/>
+
+    <if condition='!isDefined("DRSHUB_LOG_APPENDER")'>
+        <then>
+            <variable name="DRSHUB_LOG_APPENDER" value="Console-Stackdriver" />
+        </then>
+    </if>
+
+    <appender name="Console-Standard" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.classic.encoder.PatternLayoutEncoder">
+            <Pattern>%date %-5level [%thread] %logger{36}: %message%n</Pattern>
+        </encoder>
+    </appender>
+
+    <appender name="Console-Stackdriver" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder class="ch.qos.logback.core.encoder.LayoutWrappingEncoder">
+            <layout class="org.springframework.cloud.gcp.logging.StackdriverJsonLayout">
+                <includeTraceId>true</includeTraceId>
+                <includeSpanId>true</includeSpanId>
+            </layout>
+        </encoder>
+    </appender>
+
+    <if condition='isDefined("SENTRY_DSN")'>
+        <then>
+            <appender name="Sentry" class="io.sentry.logback.SentryAppender"/>
+        </then>
+    </if>
+
+    <root level="INFO">
```

Comment:
Looking up about this root element, I found: 
> The possible levels are, in order of precedence: TRACE, DEBUG, INFO, WARN and ERROR. Each level has a corresponding method that we use to log a message at that level. If a Logger isn't explicitly assigned a level, it inherits the level of its closest ancestor. The root logger defaults to DEBUG.

So this means we are logging INFO, WARN, and ERROR, but not DEBUG or TRACE right? Would we want DEBUG for local logs?

Reply:
We might want DEBUG for local logs, but you can always just edit the xml file to do so. I'm hesitant to emit DEBUG logs by default because that can result in a TOOOON of logs that we're not always interested in.